config/environments/development.rb

```diff
@@ -17,6 +17,9 @@
   # Enable server timing
   config.server_timing = true
 
+  # default url options in your environments files.
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
   if Rails.root.join("tmp/caching-dev.txt").exist?
```

Comment:
- [ ] While trying to sign up and then try to login it says `You have to confirm your email address before continuing.` and I didn't receive any confirmation mail, That's happening because you didn't configure devise mailer. So Please fix that.
[Hint] See this [article](https://www.bogotobogo.com/RubyOnRails/RubyOnRails_Devise_Authentication_Sending_Confirmation_Email.php#:~:text=Configuring%20Devise%20mailer,-So%20far%2C%20we%27re) to help you configure it right.

![image](https://user-images.githubusercontent.com/68796232/193060937-d7669103-f00b-44a2-a3cd-234b834761ff.png)